auditlog/tests/test_auditlog.py

```diff
@@ -332,6 +332,7 @@ def test_01_field_and_model_removal(self):
 
     def test_02_migration(self):
         """Test the migration of the data model related to this feature"""
+        self.env.cr.execute("""DROP VIEW auditlog_log_line_view""")
```

Comment:
I think all this test should go out, as even the migration script shouldn't be here in this branch.